getting-started.md

```diff
@@ -53,7 +53,8 @@ Also, I'm not responsible for anything I say in this guide, as [the LICENCE of t
 ## Cloning the repo
 
 After creating a repo on GitHub, the next step is to clone it.
-It means downloading a copy of the repo to your compter, so that you can edit the files in it.
+It means downloading a copy of the repo to your computer, so that you can edit the files in it.
+We will later run commands to keep your copy and GitHub in sync.
```

Comment:
```
<Akuli> you mean that after i clone something, git remembers where it was cloned from`?
<Akuli> and can push to the same place?
<Arrinao> yes
```

Reply:
It means downloading a copy of the repo to your computer, so that you can edit the files in it and synchronize them with the repo on the website.

Reply:
Is it better now? I also mentioned that `https://github.com/...` goes to `.git/config`.